docs/README.md

```diff
@@ -94,7 +94,8 @@ Now, you are able to use all blocks exported by the `modal-layout` app. Check ou
 | `disableEscapeKeyDown` | `boolean` | Whether the modal should be closed when pressing the `Esc` key (`true`) or not (`false`). | `false` | 
 | `fullScreen` | `boolean` | Whether the modal should fill the whole screen (`true`) or not (`false`). This prop is responsive i.e. it adapts itself to the device's breakpoints.  | `false` | 
 | `backdrop` | `enum` | Whether the modal will have a clickable backdrop (`clickable`) or no backdrop at all (`none`). This prop is responsive i.e. it adapts itself to the device's breakpoints. | `clickable` | 
-
+| `customPixelEventId` | `string`  | Store event ID responsible for triggering the `modal-layout` block (hence triggering the closing of `modal-layout` blocks on the interface as well). | `undefined`    |
+| `customPixelEventName` | `string`                                                                   | Store event name responsible for triggering the `modal-layout` block (hence triggering the closing of `modal-layout` blocks on the interface as well). Some examples are: `'addToCart'` and `'removeFromCart'` events. Notice that using this prop will make the associated `modal-layout` close in **every** event with the specified name if no `customPixelEventId` is specified. | `undefined`    |
```

Comment:
I'm not sure about using these same prop names for performing an action that is exactly the opposite of what they do in the `modal-trigger` block. 

Also, "triggering" the `modal-layout` block here is being used in the sense of _closing_ the modal, while it was already used before (in the `modal-trigger`'s prop description above) with the exact opposite meaning.

Also, why not add these props to `modal-actions.close` instead of the `modal-layout` itself?

Reply:
Victor! I used the modal-layout itself as the [documentation](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-using-events-to-trigger-side-effects-on-store-components) already stated that it's the one having a pixel event. And because the `modal-action` is a non mandatory block.

But anyways, we can change it if you prefer it in there!

The problem regarding the naming convention, is that it's how it works across the functionality :(

Reply:
@victorhmp what should we do?

Reply:
If I understood correctly, @victorhmp is suggesting that these props be added to `modal-actions.close`, which is equivalent to the closing icon in the modal. I agree with him that this would be more consistent.

Reply:
But you are expecting that the `modal-actions` should, from now on, be used if you want this custom pixel event. I don't think that even if this is a consistency matter, you should enforce the use of a dud block that maybe they don't want to implement.

Reply:
<img width="996" alt="" src="https://user-images.githubusercontent.com/50715158/130613152-635deee3-8394-4c3b-83c0-721231ac21df.png">

The close button will be rendered, and then the developers should go and hide it via CSS. I don't feel this is a clean implementation.

Reply:
Think it like this. The `trigger` is the one who opens the modal. And now that. the modal is open, the `modal` is the one who closes itself, the modal is mandatory. The `modal-actions` is not. Then why the trigger is not part of an action?

Reply:
That makes sense to me, @Git-the-Sanz. I feel good about merging this now, but I'd like to hear if @victorhmp has any additional remarks or suggestions to get this new behavior merged.

Reply:
Yep, makes sense to me too.